docs/requirements.txt

```diff
@@ -1,4 +1,3 @@
-sphinx
-sphinx_rtd_theme
-recommonmark
-reno[sphinx]~=4.0
+pydata-sphinx-theme==0.14.3
+sphinx==7.3.7
+reno[sphinx]==4.1.0  # keep synced with requirements.txt
```

Comment:
it's actually `reno~=4.0` in `requirements.txt`, not a fixed 4.1.0

Reply:
also, do we actually need reno for docs build now that you removed release notes section?